docs/tutorials/working-with-webassembly.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Working with WebAssembly
+---
+
+<head>
+  <link rel="canonical" href="https://docs.rancherdesktop.io/tutorials/working-with-webassembly"/>
+</head>
+
+Rancher Desktop 1.13.0 added experimental support for running WebAssembly (WASM) applications. This feature needs to be enabled in  [Preferences > Container Engine > General](../ui/preferences/container-engine/general.md).
+
+:::caution warning
+Note that when using the `moby` container engine, enabling the WASM feature switches to a different image store, so previously built or downloaded images will not be available and must be built or downloaded again. The images are not lost; Rancher Desktop will switch back to the old image store when WASM is disabled again.
+:::
+
+## Managing containerd WASM shims
+
+To run WebAssembly applications on a container runtime requires a "shim" for each WASM runtime/framework that is going to be used.
```

Comment:
```suggestion
Running WebAssembly applications on a container runtime requires a specific containerd "shim" corresponding to the WASM runtime/framework used. There are various providers such as spin, runwasi, slight etc who provide containerd shims.
```

Reply:
I've applied the first part of the suggestion. But the last sentence is not really helpful: either the user knows about this already (and therefore doesn't need the info), or they don't, in which case it doesn't tell them anything about the runtimes, the shims, where to get them etc.

Let's expand on this for the next release to have more beginner-friendly docs.